client/lcd/lcd_test.go

```diff
@@ -353,6 +353,35 @@ func TestTxs(t *testing.T) {
 	require.Equal(t, resultTx.Height, indexedTxs[0].Height)
 }
 
+func TestPoolQuery(t *testing.T) {
+	_, password := "test", "1234567890"
+	addr, _ := CreateAddr(t, "test", password, GetKeyBase(t))
+	cleanup, _, port := InitializeTestLCD(t, 1, []sdk.AccAddress{addr})
+	defer cleanup()
+
+	res, body := Request(t, port, "GET", "/stake/pool", nil)
+	require.Equal(t, http.StatusOK, res.StatusCode, body)
```

Comment:
Let's do some check on the body? Currently this test just ensures that the endpoint doesn't error, not that it actually does anything.

Reply:
@ValarDragon I'm trying to compare the initial pool with the returned one
```
expected: types.Pool{
LooseTokens:1000000000000,
BondedTokens:0,
InflationLastTime:time.Time{wall:0x0, ext:62135596800, loc:(*time.Location)(0x22aad00)}, Inflation:700000000,
DateLastCommissionReset:0,
PrevBondedShares:0}
		
actual  : types.Pool{
LooseTokens:1500019965200, 
BondedTokens:1000000000000,
InflationLastTime:time.Time{wall:0x1d6c1165, ext:63670381882, loc:(*time.Location)(nil)}, Inflation:700059763, 
DateLastCommissionReset:0, 
PrevBondedShares:0}
```

Reply:
@ValarDragon could you help me figuring out where `LooseTokens`, `BondedTokens` and `Inflation` are updated ?